README.md

```diff
@@ -1,57 +1,49 @@
 ```
+where `<conda_exec>` can be `mamba`, `micromamba`, or `conda` (not recommended).
 
 ### Downloading the model file
 
 The retrieval models can be downloaded from:
- - Version 0: [https://rain.atmos.colostate.edu/gprof_nn/chimp/chimp_smhi_v0.pt](https://rain.atmos.colostate.edu/gprof_nn/chimp/chimp_smhi_v0.pt).
- - Version 1: [https://rain.atmos.colostate.edu/gprof_nn/chimp/chimp_smhi_v1.pt](https://rain.atmos.colostate.edu/gprof_nn/chimp/chimp_smhi_v1.pt).
- - Version 2: [https://rain.atmos.colostate.edu/gprof_nn/chimp/chimp_smhi_v2.pt](https://rain.atmos.colostate.edu/gprof_nn/chimp/chimp_smhi_v1.pt).
- - Version 3: [https://huggingface.co/simonpf/chimp_smhi](https://huggingface.co/simonpf/chimp_smhi).
+ - versions `<  v3`: [https://rain.atmos.colostate.edu/gprof_nn/chimp/](https://rain.atmos.colostate.edu/gprof_nn/chimp/).
+ - versions `>= v3`: [https://huggingface.co/simonpf/chimp_smhi](https://huggingface.co/simonpf/chimp_smhi).
 
 ## Running retrievals
 
-Running CHIMP retrievals on SEVIRI files in HRIT format involves two steps: First the SEVIRI input files must be converted to the input data format expected by CHIMP. Secondly, input files must be processed using the ``chimp`` command.
-
-### Extracting the CHIMP input data
+Running CHIMP retrievals on SEVIRI files involves three steps as described further below.
 
-The ``hrit2chimp.py`` script implements a command line application to convert all SEVIRI files in a given input folder to corresponding CHIMP input files. It can be used as follows.
 
-``` shellsession
-python hrit2chimp.py <path/to/folder/with/seviri/data> <path/to/chimp/input>
+1- First, one needs to set the `CHIMP_EXTENSION_MODULES` environment variable as follows:
 ```
-
-The script combines the observations from all SEVIRI channels and writes them into a single CHIMP input file. The seviri input files are written to ``<path/to/chimp/input>/seviri`` since `chimp` expects the inputs from all sensors to be organized into respective subfolders.
-
-### Running CHIMP
-
-Assuming ``hrit2chimp.py`` has been used to write CHIMP input files to ``<path/to/chimp/input>``, the retrieval can be run using:
-
-``` shellsession
-chimp process -v <path/to/model/> seviri <path/to/chimp/input> <path/to/chimp/output> --device cpu
+export CHIMP_EXTENSION_MODULES=chimp_ext_seviri
 ```
-For the sequence-based model the process command also needs to specify the number of input steps using the ``sequence_length`` option.
+This makes the `chimp` command-line interface (CLI) aware of the `chimp_ext_seviri` module, which is needed to read the SEVIRI files.
 
-``` shellsession
-chimp process -v <path/to/model/> seviri <path/to/chimp/input> <path/to/chimp/output> --device cpu --sequence_length 16
+2- Then the `chimp_ext_seviri` module needs to be made available to and discoverable by the `chimp`. This can be done by downloading the `chimp_ext_seviri.py` module and modifying the python path accordingly, i.e.
```

Comment:
Can we make this package installable by adding eg a pyproject.toml file?

Reply:
Done!